RevenueCatUI/CustomerCenter/Views/NoSubscriptionsCardView.swift

```diff
@@ -50,12 +50,13 @@ struct NoSubscriptionsCardView: View {
 
     init(
         screenOffering: CustomerCenterConfigData.ScreenOffering?,
+        screen: CustomerCenterConfigData.Screen?,
```

Comment:
How can this be null? Maybe we should force a non-null here to simplify the code

Reply:
It can be null in the fallback screen, when there are no screen available 🤔